docs/running-headscale-container.md

```diff
@@ -44,7 +44,13 @@ curl https://raw.githubusercontent.com/juanfont/headscale/main/config-example.ya
 touch ./config/config.yaml
 ```
 
-Modify the config file to your preferences before launching Docker container.
+Modify the config file to your preferences before launching Docker container. Here are some settings that you likely want:
+```yaml
+server_url: http://your-host-name:8080 # Change to your hostname or host IP
+metrics_listen_addr: 0.0.0.0:9090 # Listen to 0.0.0.0 so it's accessible outside the container
+private_key_path: /etc/headscale/private.key # The default /var/lib/headscale path is not writable in the container
+db_path: /etc/headscale/db.sqlite # The default /var/lib/headscale path is not writable  in the container
```

Comment:
Should we not rather use a docker volume here?

/etc sounds like the wrong place

Reply:
`/etc/headscale` is the docker volume - or do you want a separate volume for the key and db? That sounds a bit overkill to me.

Reply:
I also in practice would rather go with the approach of only one volume.

Reply:
I want to separate variable data (`/var`, so database and key that changes over time) from the configuration (`/etc`)

So one way is to mount a data folder into `/var/lib/headscale` or do create a docker volume and mount it to the same folder.

Reply:
Maybe just mention it in the documentation. That either way is possible.

Reply:
@kradalby The current docs mention creating the `db.sqlite` in the config directory/volume. The only thing I fixed was the incorrect paths in the config file.
If you want to make a separate `data` directory mapped to `/var/lib/headscale` you also need to change the first steps of the doc.